tests/x11/evolution.pm

```diff
@@ -35,13 +35,15 @@ sub run {
         }
         assert_screen 'test-evolution-1';
     }
-    # Evolution 3.26 launches the main window before the new account assistant
-    if (match_has_tag('evolution-mainwindow-launched')) {
-        send_key "alt-f4";        # close the new account assistant
-        assert_screen("evolution-main-window");
+
+    #close assistant
+    send_key 'alt-f4';
+
+    assert_screen(['generic-desktop', 'evolution-main-window'], timeout => '30');
```

Comment:
Are you passing the timeout as string value?

Reply:
Might be, I actually don't know if Perl handles this as expected, but at least the code works now